ipa-core/src/protocol/ipa_prf/malicious_security/verifier.rs

```diff
@@ -7,75 +7,74 @@ use crate::{
     },
 };
 
-#[allow(non_upper_case_globals)]
-pub struct ProofVerifier<F: PrimeField> {
-    zero_share: F,
-}
-
 ///
 /// Distributed Zero Knowledge Proofs algorithm drawn from
 /// `https://eprint.iacr.org/2023/909.pdf`
 ///
 #[allow(non_upper_case_globals)]
```

Comment:
you could also define this suppression at the beginning of this module, so you don't have to repeat yourself

```rust
#![allow(non_upper_case_globals)]
```